markdown/events/2022/bytesize-gitpod.md

```diff
@@ -25,3 +25,121 @@ It is our hope that these talks / videos will build an archive of training mater
+
+[6:21](https://www.youtube.com/watch?v=kBoC6QBU-M0t=381)
+Just to show you how this is actually working, the Gitpod.yaml file is not... Did I click the right one? I'll go back. This particular repository is in the master. For some pipelines (and this is a very good example), that you want to run on Gitpod, you need to make sure it's got this Gitpod.yaml file, because this is all the programs you need. If we click on that, it's showing that we have an image, so a docker file from dockerHub. The image is being pulled, that has all the code that we need and I will go a bit more into these Gitpod yamls a bit later.
+
+[7:05](https://www.youtube.com/watch?v=kBoC6QBU-M0t=425)
+We need to be in this development branch. I also mention at this point, if you're actually making changes to nf-core code then you'll be wanting to do this in the dev branch and not on the master branch. If we click it, we should get back to a Gitpod environment, that I was expecting. That should have all the code that we need to run this particular pipeline. Sometimes this can take a few seconds. Probably because it's a live demo, it'll be more than that, but if not we can move on to different things I can explain to you. It should pop up reasonably fast. It won't? No. Okay, there we go. Now we are actually in the the correct repository that I wanted to be in, but just move that away, just to make it clearer. You can see there's a gitpod.yml file which I was expecting to see. This is the reason why we have the code brought into this environment. In the terminal, if we type in `nextflow` it finds the tool because it's already installed within this gitpod.yml file. It told us that this container contains all the things we need to run the particular piece of code.
+
+[8:18](https://www.youtube.com/watch?v=kBoC6QBU-M0t=498)
+The other thing that's quite nice here if you want to actually test this repository, there's obviously the readme. If we open this... This is obviously not rendered... There's a preview button, so we can easily click things to preview the render to look exactly the same as it would do on the web page. It also tracks, so if you're actually needing to change something in the readme you can you can add whatever you want and it will automatically change it in the preview on the right-hand side. But I'm not going to change this at the moment. If I move this across, if we look at the the readme, it tells us how we can run this piece of code and I just want to quickly show you that it's quite easy to quickly run this particular piece of code.
```

Comment:
```suggestion
The other thing that's quite nice here if you want to actually test this repository, there's obviously the readme. If we open this... This is obviously not rendered... There's a preview button, so we can easily click things to preview the render to look exactly the same as it would do on the web page. It also tracks, so if you're actually needing to change something in the readme you can add whatever you want and it will automatically change it in the preview on the right-hand side. But I'm not going to change this at the moment. If I move this across, if we look at the readme, it tells us how we can run this piece of code and I just want to quickly show you that it's quite easy to quickly run this particular piece of code.
```